server/models/Address.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+exports.addressSchema = new mongoose.Schema({
+  pickup: {
+    type: String,
+    required: true,
+    default: null,
+  },
+  service: {
+    type: String,
+    required: true,
+    default: null,
+  },
+  dropoff: {
+    type: String,
+    default: function () {
+      if (this.pickup) {
+        return this.pickup;
+      }
+
+      return null;
+    },
```

Comment:
Wouldn't this just be unnecessary? Since we're just using the pickup address?

Reply:
Well, it's more-so to allow people to specify a different dropoff location, but I figured most would likely just have the dropoff be the same as the pickup. So, most of the time it wouldn't be necessary. But, if someone wanted to pick a different dropoff, the option is there.